app/server/main.ts

```diff
@@ -4,223 +4,248 @@ import "./methods/task/task-methods";
+                "https://cdn.pixabay.com/photo/2015/04/23/22/00/tree-736885_1280.jpg",
+                "https://cdn.pixabay.com/photo/2024/05/26/10/15/bird-8788491_1280.jpg",
+            ],
+            inspection_ids: ["1", "2"],
+        });
+
+        ListingStatusCollection.insertAsync({ _id: "1", name: "Draft" });
+    }
 }
 // This function is used to seed the database with initial task data
 async function tempSeedTaskData(): Promise<void> {
-  if ((await TaskCollection.find().countAsync()) === 0) {
-    TaskStatusCollection.insertAsync({
-      _id: "1",
-      name: "Not Started",
-    });
-
-    TaskStatusCollection.insertAsync({
-      _id: "2",
-      name: "In Progress",
-    });
-
-    TaskStatusCollection.insertAsync({
-      _id: "3",
-      name: "Completed",
-    });
-
-    TaskCollection.insertAsync({
-      _id: "1",
-      name: "Initial listing meeting",
-      taskStatus: TaskStatus.NOTSTARTED,
-      createdDate: new Date("2025-04-12T10:00:00Z"),
-      dueDate: new Date("2025-04-19T10:00:00Z"),
-      description:
-        "Meet with the client to discuss the property listing process and gather necessary information.",
-      priority: "High",
-    });
-    TaskCollection.insertAsync({
-      _id: "2",
-      name: "Follow-up with client",
-      taskStatus: TaskStatus.INPROGRESS,
-      createdDate: new Date("2025-04-20T10:00:00Z"),
-      dueDate: new Date("2025-04-27T10:00:00Z"),
-      description:
-        "Check in with the client to provide updates and address any questions.",
-      priority: "Medium",
-    });
-  }
+    if ((await TaskCollection.find().countAsync()) === 0) {
+        TaskStatusCollection.insertAsync({
+            _id: "1",
+            name: "Not Started",
+        });
+
+        TaskStatusCollection.insertAsync({
+            _id: "2",
+            name: "In Progress",
+        });
+
+        TaskStatusCollection.insertAsync({
+            _id: "3",
+            name: "Completed",
+        });
+
+        TaskCollection.insertAsync({
+            _id: "1",
+            name: "Initial listing meeting",
+            taskStatus: TaskStatus.NOTSTARTED,
+            createdDate: new Date("2025-04-12T10:00:00Z"),
+            dueDate: new Date("2025-04-19T10:00:00Z"),
+            description:
+                "Meet with the client to discuss the property listing process and gather necessary information.",
+            priority: "High",
+        });
+        TaskCollection.insertAsync({
+            _id: "2",
+            name: "Follow-up with client",
+            taskStatus: TaskStatus.INPROGRESS,
+            createdDate: new Date("2025-04-20T10:00:00Z"),
+            dueDate: new Date("2025-04-27T10:00:00Z"),
+            description:
+                "Check in with the client to provide updates and address any questions.",
+            priority: "Medium",
+        });
+    }
+}
+async function tempSeedProfileData(): Promise<void> {
```

Comment:
Is this profile data connected to any user or anything ? cos profile data is dependant on a user creation. 
so I think this could be actually unnecessary, because earlier we already registering a user with some of the common profile data so rows in profile data class will always exist. if you want to populated other fileds such as dob and etc then you can use the profile_update method. but again not necessary. cause in real life when someone registers they wont fill out the specific details until they decide to edit profile